First attempt at an improved design

ndw opened the pull request:
This PR redesigns the site to make links easier to access and to tighten up the front page text. (There will be another round of editorial proposals later.)

You can preview the new design at https://mdecltest.nwalsh.com/

dalapeyre commented:
Nice look. I will try to give more feedback
later, busy today.


Should calendar and resources be separated?

I would not consider a calendar a resource,
but that might just be me.

—dal

> On Apr 28, 2019, at 3:53 PM, Norman Walsh <notifications@github.com> wrote:
> 
> @ndw requested your review on: #44 First attempt at an improved design.
> 
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 


================================================================
Deborah A Lapeyre              mailto:dalapeyre@mulberrytech.com
Mulberry Technologies, Inc.      http://www.mulberrytech.com
17 West Jefferson Street         Phone: 301-315-9631 (USA)
Suite 207                        Fax:   301-315-8385
Rockville, MD 20850
----------------------------------------------------------------
Mulberry Technologies: Consultancy for XML, XSLT, and Schematron
================================================================

ndw commented:
I was trying to limit navigation items to a small, fixed set. I think that's going to lump a few things under "resources" until the site gets bigger. However, I kind of like making events more prominent, so I added that to the sidebar on the home page.

Other changes in this most recent commit:

1. Fix some pedantic issues about the JS used for the "?" links on the home page. Now they will not appear to be links if you're using a screen reader or non-JS enabled browser.
2. I reworked the navigation links on the secondary pages so that the links are always below the title. It was weird before.
